src/main/java/seedu/duke/WellNus.java

```diff
@@ -1,11 +1,15 @@
+
 package seedu.duke;
 
+import seedu.duke.atomichabit.atomichabit.AtomicHabitManager;
+
 import java.util.Scanner;
 
 public class WellNus {
     /**
      * Main entry-point for the java.duke.Duke application.
      */
+
```

Comment:
Redundant blank line

Reply:
Resolved in 3371f317f0f77df8dbf6210446555e6af3ebac85.